### PR TITLE
AST-1546 - WooCommerce 6.1 Products attributes are center aligned

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description
Forum*
https://wordpress.org/support/topic/products-attributes-unaligned-with-astra-theme-and-woocommerce-6-1/

*PR from the WooCommerce:*

https://github.com/woocommerce/woocommerce/pull/31105/files

On our end, we are actually overriding the WooCommerce CSS.
So, in WooCommerce 6.1.0 the product variation table label column text is aligned to the center when the Astra theme is active.

### Screenshots
- https://share.bsf.io/Qwu4E2z4

### Types of changes

 Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
Added in the card.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by imnavanath — APPROVED** [review]

File: sass/site/compatibility/woocommerce/woocommerce.scss [hunk 1]
```diff
@@ -3914,3 +3914,14 @@ button.pswp__button--zoom:hover {
...
+/**
+ * Product variation - label align css.
+ */
+ .woocommerce {
+	.variations {
+		th {
+			text-align: left;
+		}
+	}
+}
```
Comment (premanshup) [inline sass/site/compatibility/woocommerce/woocommerce.scss:3927]: @GauravKhupse We can move this CSS to already defined .woocommerce at line number 423 of the same file.
```
/**
 * Main WooCommerce styles
 */
.woocommerce {
```
Also, why are we adding the same CSS in other file? Only adding CSS here should work?

**Review by premanshup — CHANGES_REQUESTED** [review]

File: sass/site/compatibility/woocommerce/woocommerce.scss (hunk 1 above)
Comment (GauravKhupse) [reply sass/site/compatibility/woocommerce/woocommerce.scss:3933]: Changes done!

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css [hunk 2]
```diff
@@ -2208,6 +2208,10 @@
     padding: 14px;
 }
 
+.astra-popover-color .components-h-stack .components-button {
+    flex: inherit;
+}
+
```
Comment (premanshup) [inline inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css:2214]: @GauravKhupse This change should not be here, can you please confirm?

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css (hunk 2 above)
Comment (GauravKhupse) [reply inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css]: Removed unwanted CSS. changes done

**Review by premanshup — DISMISSED** [review]

**Pull request merged**